rtt/transports/corba/RemotePorts.cpp

```diff
@@ -194,6 +189,20 @@ RTT::base::ChannelElementBase::shared_ptr RemoteInputPort::buildRemoteChannelOut
     return corba_ceb;
 }
 
+bool RemoteInputPort::disconnect(PortInterface* port)
+{
+    //just check the type and not relay on
+    RemoteOutputPort *oPort = dynamic_cast<RemoteOutputPort* >(port);
+    if(oPort == NULL){
+        Logger::In in("RemoteInputPort::disconnect(PortInterface& port)");
+        log(Error) << "Port: " << this->getName() << " could not be disconnected from: " << oPort->getName()
+                << " because it is not of type: RemoteOutputPort" << endlog();
+        return false;
+    }
+
+    return dataflow->removeConnection(oPort->getName().c_str(), this->getDataFlowInterface(), this->getName().c_str());
```

Comment:
@doudou, true you are right, could be simplified by just using "dataflow" here.

Actually I was simply not sure if the port has to be Remote, It was a guess based on the other implementations. So would it allow to be a local port as well by using.

Also just wondering if 
`return port.disconnect(this)`
could do the job here for all cases?